### PR TITLE
rfc5424-logging-handler: new recipe

### PR DESCRIPTION
**Pull request by tschoonj**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/rfc5424-logging-handler/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+test:
+  imports:
+    - rfc5424logging
+
+about:
+  home: https://github.com/jobec/rfc5424-logging-handler
+  license: BSD
+  license_family: BSD
+  license_file: LICENSE
```
Comment (marcelotrevisani) [inline recipes/rfc5424-logging-handler/meta.yaml]: The license file is not being packaged. Could you please raise this issue on this project?
For now you can add it manually

Comment (tschoonj) [reply recipes/rfc5424-logging-handler/meta.yaml]: How does one do that? `curl` in the LICENSE file from the repo?

Comment (tschoonj) [reply recipes/rfc5424-logging-handler/meta.yaml]: Fixed it by specifying it as an additional source

**Comment by tschoonj** [comment]
@marcelotrevisani this is one is ready for a merge I think...

File: recipes/rfc5424-logging-handler/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    - pytz
+    - tzlocal
+
+test:
+  imports:
+    - rfc5424logging
+
+about:
+  home: https://github.com/jobec/rfc5424-logging-handler
+  license: BSD
```
[the remark this answers is not in the document]
Comment (tschoonj) [reply recipes/rfc5424-logging-handler/meta.yaml]: BSD 3-Clause, fixed in recipe

**Pull request merged**